### DISCUSSION TITLE
Branch searching in branch listing (Pro)

### DISCUSSION THREAD

**Discussion by joshuapretorius** [six7/figma-tokens#1898] [Ideas]
We have a lot of GitHub branches as part of our workflow, and I needed to recover a .json file from a specific branch and the only way for me to find the right branch was to scroll through a list 1000s of branches and I luckily spotted it after around 10mins of scrolling. 

**Proposal:** A search and select feature as part of the GitHub branch listing

**Comment by six7** [comment]
Already [on our roadmap](https://github.com/tokens-studio/figma-plugin/issues/1326) 👍

**Reply by joshuapretorius** [reply]
Thanks @six7. Roadmap now bookmarked ✅
